aten/src/ATen/native/SpectralOpsUtils.h

```diff
@@ -62,4 +63,18 @@ inline int64_t infer_ft_complex_to_real_onesided_size(int64_t complex_size,
   }
 }
 
+using fft_fill_with_conjugate_symmetry_fn =
+    void (*)(ScalarType dtype, IntArrayRef mirror_dims, IntArrayRef half_sizes,
+             IntArrayRef in_strides, const void * in_data,
+             IntArrayRef out_strides, void * out_data);
+DECLARE_DISPATCH(fft_fill_with_conjugate_symmetry_fn, fft_fill_with_conjugate_symmetry_stub);
+
+// In real-to-complex transform, cuFFT and MKL only fill half of the values
+// due to conjugate symmetry. This function fills in the other half of the full
+// fft by using the Hermitian symmetry in the signal.
+// self should be the shape of the full signal and dims.back() should be the
+// one-sided dimension.
```

Comment:
What is "the one-sided dimension"?

Reply:
What's the purpose of this function / how is it intended to be used?

Reply:
For two-sided real-to-complex FFTs, the mkl and cufft backends both write onesided FFT results into half of the two-sided result tensor, then this function is called to fill in the other side in-place. By one-sided dimension, I mean the dimension for which only the positive frequency terms are given.